.github/workflows/update_gdocs_data.yml

```
@@ -29,7 +29,6 @@ jobs:
           commit-message: 'chore: update docs'
           title: Update Google Docs Meta Data
           labels: chore
-          reviewers: krivard
-          assignees: krivard
+          assignees: melange396
```

did you wanna keep the commented/empty `reviewers` field?
```suggestion
          # reviewers:
          assignees: melange396
```